src/main/java/teammates/common/datatransfer/attributes/FeedbackQuestionAttributes.java

```diff
@@ -159,22 +159,19 @@ public String getJsonString() {
     public List<String> getInvalidityInfo() {
         FieldValidator validator = new FieldValidator();
         List<String> errors = new ArrayList<String>();
-        String error;
 
-        error = validator.getInvalidityInfoForFeedbackSessionName(feedbackSessionName);
-        if (!error.isEmpty()) {
-            errors.add(error);
-        }
+        addNonEmptyError(validator.getInvalidityInfoForFeedbackSessionName(feedbackSessionName), errors);
 
-        error = validator.getInvalidityInfoForCourseId(courseId);
-        if (!error.isEmpty()) {
-            errors.add(error);
-        }
+        addNonEmptyError(validator.getInvalidityInfoForCourseId(courseId), errors);
 
-        error = validator.getInvalidityInfoForEmail(creatorEmail);
+        // special case when additional text should be added to error text
+        String error = validator.getInvalidityInfoForEmail(creatorEmail);
         if (!error.isEmpty()) {
-            errors.add("Invalid creator's email: " + error);
+            StringBuffer buffer = new StringBuffer();
+            buffer.append("Invalid creator's email: ").append(error);
+            error = buffer.toString();
```

Comment:
Thanks, looks cleaner without need for StringBuffer variable, so I corrected in my code.